api/src/test/java/org/openmrs/module/adminui/utils/RegexUtilTest.java

```diff
@@ -0,0 +1,57 @@
+package org.openmrs.module.adminui.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RegexUtilTest {
+
+	@Test
+	public void shouldEvaluateToFalseForStringWithNoDigit(){
+		RegexUtil util = new RegexUtil("thisissomething", RegexUtil.MUST_CONTAIN_DIGIT_REGEX);
+		Assert.assertFalse("String should have at least a digit", util.matcher());
+	}
+
+	@Test
+	public void shouldEvaluateToTrueForStringWithDigits(){
+		RegexUtil util = new RegexUtil("thisis12something", RegexUtil.MUST_CONTAIN_DIGIT_REGEX);
+		Assert.assertTrue("Could not match string with at least a digit", util.matcher());
+	}
+
+	@Test
+	public void shouldEvaluateToTrueForStringWithDigitsAndSpecialChars(){
+		RegexUtil util = new RegexUtil("thisis12something@#", RegexUtil.MUST_CONTAIN_DIGIT_REGEX);
```

Comment:
This is not exactly what the test method says, it says digits and special characters only but i see 'thisis' and 'something' in it